examples/19_file_reader/main.neva

```diff
@@ -0,0 +1,16 @@
+import { io }
+
+const filename string = '19_file_reader/main.neva'
+
+component Main(start any) (stop any) {
+  nodes {
+    io.FileReader
```

Comment:
how about naming it `ReadAll` like in Go since we're planning to add more flexible version in the future?

Also, I suggest using verbs e.g. ReadFile instead of FileReader (naming convention was changed in v0.16)